readme.md

```diff
@@ -327,9 +327,7 @@ Cards
 
 ### Creating a card
 
-When creating a card you can use either a card or a token
-
-With a token:
+When creating a card you can use either a card or a token. For [PCI compliance](https://stripe.com/docs/security#validating-pci-compliance) though we highly recommend that you pass a token generated client-side.
```

Comment:
Maybe follow this up with "... so that raw credit card numbers are never directly handled by your servers".